src/GDGUkraine/rest_controller.py

```diff
@@ -511,6 +511,22 @@ def generate_invites(self, id):
             raise HTTPError(500, "Cannot save generated invites") from e
         return {"ok": True}
 
+    @cherrypy.tools.json_out()
+    @cherrypy.tools.authorize()
+    def record_visit(self, id):
+        '''POST /api/events/:id/check-in'''
+        req = cherrypy.request
+        orm_session = req.orm_session
+        reg_id = int(id)
+        reg_data = api.get_event_registration_by_id(orm_session, reg_id)
+        visit = EventParticipant(
+            visited=True
+            )
+        if reg_data:
```

Comment:
I'd `raise HTTPError(400, msg)` with error explanation instead, with `if not reg_data` check.